package.json

```diff
@@ -148,6 +147,7 @@
     "node-sass": "^4.9.3",
     "rimraf": "^2.6.3",
     "sass-loader": "^6.0.7",
+    "semantic-ui-react": "^0.88.2",
```

Comment:
'semantic-ui-react' should be listed in the project's dependencies, not devDependencies